oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/TableMetaInfo.java

```diff
@@ -54,15 +54,25 @@ public static void addModel(Model model) {
             storageAndColumnMap.put(columnName.getStorageName(), columnName.getName());
         });
 
+        storageAndTagMap.put(InstanceTraffic.NAME, InfluxConstants.TagName.NAME);
         if (model.getName().endsWith("_traffic")) {
-            // instance_traffic service_id, endpoint_traffic service_id
-            if (InstanceTraffic.INDEX_NAME.equals(model.getName())
-                || EndpointTraffic.INDEX_NAME.equals(model.getName())) {
-                storageAndTagMap.put(EndpointTraffic.SERVICE_ID, InfluxConstants.TagName.SERVICE_ID);
-            } else {
+            switch (model.getName()) {
+                // instance_traffic name, service_id
+                case InstanceTraffic.INDEX_NAME: {
+                    storageAndTagMap.put(InstanceTraffic.NAME, InfluxConstants.TagName.NAME);
```

Comment:
`InstanceTraffic#Name` is a big data set, are you sure you need this as a tag?

Reply:
`MetadataQuery#getServiceInstances` dependent on `NAME` to avoid duplication records.